db/db.py

```diff
@@ -0,0 +1,28 @@
+from dotenv import load_dotenv
```

Comment:
consider adding a `requirements.txt` to the repository to have people install `dotenv` package.

Reply:
Added